hydra-configs-torch/hydra_configs/torch/nn/modules/__init__.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+# flake8: noqa
+from .loss import BCELossConf
+from .loss import BCEWithLogitsLossConf
+from .loss import CosineEmbeddingLossConf
+from .loss import CTCLossConf
+from .loss import L1LossConf
+from .loss import HingeEmbeddingLossConf
+from .loss import KLDivLossConf
+from .loss import MarginRankingLossConf
+from .loss import MSELossConf
+from .loss import MultiLabelMarginLossConf
+from .loss import MultiLabelSoftMarginLossConf
+from .loss import MultiMarginLossConf
+from .loss import NLLLossConf
+from .loss import NLLLoss2dConf
+from .loss import PoissonNLLLossConf
+from .loss import SmoothL1LossConf
+from .loss import SoftMarginLossConf
+from .loss import TripletMarginLossConf
```

Comment:
feels like this would better be inside loss.py and `__init__.py` would just call it from its register function.

Reply:
It makes sense there, but `loss.py` is a configen output. This is manual code.

Reply:
maybe we should create a convention:
```
loss.py # generated
loss_handcoded.py # manual additions.
```

At this point, the `__init__.py` file can import them both (and possibly also export from both, making them appear like one file).

Reply:
not a bad idea. I think we need a convention for handling manual configs anyways.

Reply:
not a bad idea. I think we need a convention for handling manual configs anyways.

Reply:
here is an even better idea.
We can add support to custom import to configen at the top and the bottom of the generated configs.
Allowing automatic import of user provided files if they exist.

Reply:
Could you elaborate on this?

Do you mean chain the 'manual' file to the `generated one` through importing that is automatically added by configen if a field is passed in the configen conf yaml?

Reply:
not exactly.
I mean importing if a file with a specific name exists.

generated/foo.py
```python
try:
 from . import foo_header
except ImportError:
  pass

# generated code here

try:
 from . import foo_footer
except ImportError:
  pass

```